paper.tex

```diff
@@ -215,7 +215,7 @@ \subsection{Trading}
     \label{fig:trading_product}
 \end{figure}
 
-Next, session key $k$ should be exchanged between the data provider and consumers. To exchange data, Azaria et al\cite{Medrec} seek an off-chain solution which is based on an end-to-end communication to ensure the efficiency and low costs. However, it is hard to avoid that the data source is unavailable or the sender sends incorrect data intentionally. Instead of using off-chain solution, we use smart contract to transfer the session key\cite{3tierDataMarket} that not only ensures the consistency of the session key, the availability of the source and the traceability of the record, but also prevents malicious participants to cheat others.
+In the proposed architecture, the "access" to specific data stream is traded instead of giving out the raw data copies. Subscribers can retreive data on MAM at any time without equiping extra storage to preserve data, and one can also query for arbitrary section of data rather downloading them all. Thus, the most important part of trading process is to give the session key $k$, the encryption key of data stream, to the subscribers. In order to exchange data, Azaria et al\cite{Medrec} seek an off-chain solution which is based on an end-to-end communication to ensure the efficiency and low costs. However, it is hard to avoid that the data source is unavailable or the sender sends incorrect data intentionally. Instead of the off-chain solution, we use smart contract to transfer the session key\cite{3tierDataMarket} that not only ensures the consistency of the session key, the availability of the source and the traceability of the record, but also prevents malicious participants to cheat others. With the help of brokers and smart contracts, both providers and subscribers do not need to be online at the same time to procceed the trading process.
```

Comment:
subscribers do not need to be online at the same time to "proceed" the trading process.